pages/vpc/troubleshooting/connectivity-tests.mdx

```diff
@@ -0,0 +1,95 @@
+
+1. [Connect to another Instance](/instances/how-to/connect-to-instance/) in the same Private Network as the non-communicating Instance.
+
+2. Try to reach the non-communicating Instance with a ping via `ping instance-name` (where `instance-name` is the name you gave your Instance upon creation.)
+
+3. Carry out the same test for both IPv4 and IPv6, ie `ping -4 instance-name` and `ping -6 instance-name`.
+    
+    - If these tests fail, there may be a DNS issue meaning the Instance's hostname cannot be resolved, see step 4 directly below.
+    - Otherwise, if the tests succeed, continue to the section [Verify ARP table](#step-2-verify-artp-table)
+    
+4. Consider checking DNS records via the Scaleway Domains and DNS CLI as documented [here](/vpc/reference-content/dns/#troubleshooting), to see if the private DNS zone is correctly configured and the Instance's DNS records are up-to-date. 
+
+    <Message type="important">
+    Note that we do not recommend editing these records yourself, rather open a support ticket with the information you have gathered.
+    </Message>
+
+### Step 2: Check ARP
+
+A server's **A**ddress **R**esolution **P**rotocol (ARP) table maps IP addresses to MAC addresses on a local network. Use this data to find the probe state for the target Instance's MAC address.
+
+1.  From another Instance in the same Private Network as the non-communicating Instance, view ARP information via an `ip neigh` command.
+
+    You should see an output of many lines in the following format:
+
+    ```
+    172.16.8.2 dev ens5 lladdr 02:00:00:1c:0d:9b REACHABLE
+    ```
+
+2. Find the line beginning with the non-communicating Instance's private IP address, and containing also its MAC address (you can check both via the Scaleway console in the Private Network's **Attached resources** tab). 
+
+    The line ends with the state of the probe for that Instance's MAC address.
+
+    `FAILED` shows that the system tried to resolve the MAC address but did not get a response.
+    
+    Continue to the next stage of testing.
+
+### Step 3: Try reaching the Instance using its LLA
+
+If the Instance could not be reached via the resolution of its MAC address, next you should try to reach it via its IPv6 **L**ink **L**ocal **A**ddress (LLA) in the Private Network.
+
+The LLA is calculated from the Instance's MAC address.
+
+1. Use an online tool or guide to carry out the standardized process of calculating the Instance's IPv6 LLA address from its MAC address. Remember that you can view its MAC address from the Scaleway console, in the Private Network's **Attached Resources** tab.
+
+2. Ping the LLA. Ensure that you specify which interface to ping over (the Private Network) by adding the `%interface` zone index (usually `%ens5` for a Scaleway Private Network) to the end of the ping command. For example, for a LLA of `fe80::0:0ff:fe1c:d9b`, use the following command:
```

Comment:
Shall we help them find the right interface using something like the following command?
The interface name might differ and depends on the OS as well as the Instance type. In my particular case it is a COPARM running Ubuntu 24.04.

```bash
root@scw-par:~# ip -brief -color addr show
lo               UNKNOWN        127.0.0.1/8 ::1/128
enp0s1           UP             fe80::dc00:ff:fe8f:40df/64
enp1s0           UP             192.168.0.3/22 metric 50 fdc0:aa61:494b:fa16:c5e1:f842:11ff:efab/128 fe80::ff:fe13:38d1/64
```